StitchCore/StitchCoreTests/AuthTests.swift

```diff
@@ -124,16 +124,19 @@ class AuthTests: XCTestCase {
     
     func testMultipleLoginSemantics() throws {
         let exp = expectation(description: "multiple logins work as expected")
-        let mlsStitchClient = StitchClient(appId: "stitch-tests-ios-sdk-jjmum")
         var anonUserId = ""
         var emailUserId = ""
-        
-        // check storage
-        XCTAssertFalse(mlsStitchClient.isAuthenticated)
-        XCTAssertNil(mlsStitchClient.loggedInProviderType)
-        
+        var mlsStitchClient: StitchClient!
```

Comment:
Not sure why these changes are showing up here. This was stuff from my multiple login semantics commit that made it conform to the factory changes. Can we make sure we rebase everything properly before merging?

Reply:
Odd. Alright.